blazingcache-core/src/main/java/blazingcache/network/netty/NettyChannel.java

```diff
@@ -48,15 +48,15 @@ public class NettyChannel extends Channel {
     private static final boolean DISCONNECT_ON_PENDING_REPLY_TIMEOUT = Boolean.parseBoolean(System.getProperty("blazingcache.nettychannel.disconnectonpendingreplytimeout", "true"));
     private volatile SocketChannel socket;
     private static final Logger LOGGER = Logger.getLogger(NettyChannel.class.getName());
-    private static final AtomicLong idGenerator = new AtomicLong();
+    private static final AtomicLong ID_GENERATOR = new AtomicLong();
 
     private final Map<String, ReplyCallback> pendingReplyMessages = new ConcurrentHashMap<>();
     private final Map<String, Message> pendingReplyMessagesSource = new ConcurrentHashMap<>();
     private final Map<String, Long> pendingReplyMessagesDeadline = new ConcurrentHashMap<>();
     private final ExecutorService callbackexecutor;
     private final NettyConnector connector;
     private volatile boolean ioErrors = false;
-    private final long id = idGenerator.incrementAndGet();
+    private final String id = UUID.randomUUID().toString();

```

Comment:
The only use of this id is to be printed in the toString.
idGenerator was previously used only for this id, so I repurposed the idGenerator for the messages and converted this id to a uuid (in the JVMChannel, the id field was already an uuid)